packages/models/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@quality-metrics/models",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "./index.js",
+  "types": "./src/index.d.ts",
```

Comment:
These shouldn't be necesary, the build automatically overwrites them anyway (see [esbuild.config.js](https://github.com/flowup/quality-metrics-cli/blob/ae63670d5b87ca5d80b627ccda481e835e7d3ef2/esbuild.config.js#L32-L34)).

```suggestion
```